.github/workflows/test-dotnet.yml

```diff
@@ -22,5 +22,6 @@ jobs:
 
       - name: run tests
         run: |
-          make
+         echo "No tests yet"
+         exit 1
```

Comment:
Please add something sensible here.

Reply:
i've modified that to invoke dotnet test